droid-hal-device.inc

```diff
@@ -435,6 +435,13 @@ KERNEL_RELEASE=$(cat out/target/product/%{device}/*/*/include/config/kernel.rele
 cp out/target/product/%{device}/kernel $RPM_BUILD_ROOT/boot/kernel-$KERNEL_RELEASE
 cp out/target/product/%{device}/obj/ROOT/hybris-boot_intermediates/boot-initramfs.gz $RPM_BUILD_ROOT/boot/
 
+echo "/boot/kernel-$KERNEL_RELEASE" > kernel.files
+echo "/boot/boot-initramfs.gz" >> kernel.files
+
+if cp out/target/product/%{device}/dt.img $RPM_BUILD_ROOT/boot/; then
```

Comment:
Doesn't this cause errors to be printed if not present? I would either redirect to &> /dev/null or do -f check before cp.